arrows/pdal/pointcloud_io.h

```diff
@@ -12,25 +12,30 @@
 
 #include <vital/types/local_geo_cs.h>
 #include <vital/types/landmark_map.h>
+#include <vital/algo/pointcloud_io.h>
 
 namespace kwiver {
 namespace arrows {
 namespace pdal {
 
-/// Write landmarks to a file with PDAL
-KWIVER_ALGO_PDAL_EXPORT
-void
-save_point_cloud_las(vital::path_t const& filename,
-                     vital::local_geo_cs const& lgcs,
-                     vital::landmark_map_sptr const& landmarks);
-
-/// Write point cloud to a file with PDAL
-KWIVER_ALGO_PDAL_EXPORT
-void
-save_point_cloud_las(vital::path_t const& filename,
-                     vital::local_geo_cs const& lgcs,
-                     std::vector<vital::vector_3d> const& points,
-                     std::vector<vital::rgb_color> const& colors = {});
+class KWIVER_ALGO_PDAL_EXPORT pointcloud_io
+  : public vital::algo::pointcloud_io
+{
+public:
+  PLUGIN_INFO( "pdal",
+               "Use PDAL to write pointclouds in .las format." );
```

Comment:
We might want to just want to say "Use PDAL to write point clouds" in anticipation of generalizing to other formats, which should be really easy, but not needed in this PR.

Reply:
FYI I think it is as simple as creating a kwiver config variable on this algorithm that takes a string and uses it instead of "writers.las" in 
```c++
::pdal::Stage *writer = factory.createStage("writers.las");
```
some additional error checking will be needed as well to make sure the writer is valid.  Options are here: https://pdal.io/stages/writers.html

Again, let's not do this in this PR.